README.rst

```diff
@@ -72,6 +72,23 @@ If you don't want the errors, then make use of an extra argument to the command
 
 This last step should be performed every time you create new or make changes to existing programs.
 
+Show Programs Tab
+~~~~~~~~~~~~~~~~
+
+In order to show programs tab in the LMS dashboard, users will need to manually create an entry in ``Programs api config`` model in LMS Admin Panel. Go to ``Home > Programs >  Programs api config``. Add ``Marketing path`` equal to ``/programs`` and enable it. Then Programs tab will be shown on the LMS where users can view their registered programs. It will show like as in below picture.
+
+.. image:: https://private-user-images.githubusercontent.com/4692377/270365524-4a715fbb-9c66-4b1c-a3c9-f42983fb222b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0NjI2MzEsIm5iZiI6MTcwMjQ2MjMzMSwicGF0aCI6Ii80NjkyMzc3LzI3MDM2NTUyNC00YTcxNWZiYi05YzY2LTRiMWMtYTNjOS1mNDI5ODNmYjIyMmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTNUMTAxMjExWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTY0YTgxZTIyZjA0MTRhMWIxZDRlZGM2OTY4ZmQxMmNlZTBlYjc5NjczZjFiMGU2MGIyYmIwMTNmYWM0ZmQ1MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.vzPYOlqeLPg2gN1k-1H_u2L0hcdxjzUWRtCDUxleN3U
+    :alt: Programs Tab on LMS dashboard
+
+In the above image, User can see explore programs button which is pointing to ``http://localhost:8080/programs`` by default. This link does not exist. So, users can change this link to their custom build marketing site url to show all programs their. This can be done by modifying ``Site Configurations`` model in LMS Admin Panel. Go to ``Home >  Site_Configuration › Site configurations``. Open respective LMS site configuration and add below dict in ``site values`` field like below image::
+
+    "MKTG_URLS": {
+        "ROOT": "Marking site link here"
```

Comment:
[clarity]: Is marketing site link here supposed to be LMS link?

Reply:
It can be any link that can integrate LMS/discovery APIs to show programs.

Reply:
So it should be something like `"ROOT": "https://marketing-site-here.com"`